### ISSUE TITLE
Will not run under python 3.5 since 3.3.2

### ISSUE THREAD

**Issue by alexrecarey** [pynamodb/PynamoDB#593]
After upgrading to 3.3.3 from 3.3.0, found that my app was broken running under python 3.5 which should not have happened for a point release. After checking the releases, it seems due to the file `mypy_helpers.py` in the test directory using python 3.6 code on line 15:

    with open(f'{tempdirname}/__main__.py', 'w') as f:

Thanks

**Comment by ikonst** [comment]
@alexrecarey I used newer syntax in this file since it's used only for Python\>\=3.6 mypy runs. Any insight as to how the test-only file was loaded during a non-test run?

**Comment by alexrecarey** [comment]
Not really, that's a good question. I checked the repo and nothing seems to import `mpypy_helpers.py` other than `test_mypy.py`, and `test_mypy.py` is not imported from anywhere. I will create a minimum case flask server with pynamodb to see if it is something special with my environment.

**Comment by ikonst** [comment]
Thanks!
Perhaps you have some code that traverses a package and imports all modules? :)

**Comment by alexrecarey** [comment]
I'm using gevent together with `monkey.patch_all()`. I think this has to be it. No other package I use should be doing something like that.

**Comment by ikonst** [comment]
I can't see how `monkey.patch_all()` would do that.

Can you add an
```
import pdb
pdb.set_trace()
```
to the beginning of that file (you can delete all other lines) and see the stack trace when it's imported?

**Comment by garrettheel** [comment]
@ikonst perhaps we should ensure that the `tests` directory doesn't end up in the wheel? looks like we dump it in there right now

**Comment by ikonst** [comment]
🙀 I guess we should :/

**Comment by ikonst** [comment]
@alexrecarey Can you help our investigation by adding some diagnostic code to your installation?

**Comment by alexrecarey** [comment]
Sure, let me know what info you want. I worked around the issue by creating a docker container with python 3.6 instead of deploying it to ubuntu directly, but I don't mind going back and retesting.

**Comment by ikonst** [comment]
@alexrecarey 

Can you add an
```
import pdb
pdb.set_trace()
```
to the beginning of that file (you can delete all other lines) and see the stack trace when it's imported?
